src/app/donation-start/donation-start.component.scss

```diff
@@ -296,27 +296,46 @@ a {
 
 .icons {
   display: flex;
-  flex-wrap: wrap;
+  flex-wrap: nowrap;
+  flex-direction: column;
+
+  // align icon text column on small devices
+  .icon-wrapper:first-child span {
+    padding-left: 0;
+  }
+
+  @media#{$breakpoint-md} {
+    flex-direction: row;
+
+    .icon-wrapper:first-child span {
+      padding-left: 10px;
+    }
+
+    .icon-wrapper:first-child {
+      padding-left: 0;
+    }
+  }
```

Comment:
Looks like there's some duplication between this and the camaign details css. Do you think it would be worth de-duplicating, maybe moving the shared scss into `_mixins.scss` ? Not sure.